stdlib/asm/crypto/dsa/rpo_falcon512.masm

```diff
@@ -112,39 +112,39 @@ end
 # =============================================================================================
 
 #! For an element `tau := (tau0, tau1)` in the quadratic extension field, computes all its powers
-#! `tau^i` for `i = 0,..., 512` and stores them in the memory region `[tau_ptr, tau_ptr + 513)`.
-#! The procedure returns `tau_ptr + 513`.
+#! `tau^i` for `i = 0,..., 512` and stores them in the memory region `[tau_ptr, tau_ptr + 513*4)`.
+#! The procedure returns `tau_ptr + 513*4`.
 #!
 #! Input: [tau1, tau0, tau_ptr, ...]
-#! Output: [tau_ptr + 513, ...]
+#! Output: [tau_ptr + 513*4, ...]
 #!
 #! Cycles: 8323
 export.powers_of_tau
```

Comment:
Similar comment as above re cycle count: I think this now takes at least 513 cycles more.

Reply:
Updated to ~8850